examples/the-main-function/the_main_function.bal

```diff
@@ -1,9 +1,9 @@
 import ballerina/io;
 
-// The `main` function, that performs a math operation. The first parameter `operation`,
+// The `main` function performs a math operation. The first parameter `operation`
 // represents the math operation while the second parameter `initialValue` is a defaultable
-// parameter with the default value `0`, representing the base value upon which the operation
-// should be performed. The rest parameter `values` represents the additional arguments.
+// parameter with the default value `0` representing the base value upon which the operation
+// should be performed. The other parameter `values` represents the additional arguments.
```

Comment:
rest parameter is a kind of parameter in Ballerina (e.g., int...).

So rest is OK IMO.

```suggestion
// should be performed. The rest parameter `values` represents the additional arguments.
```